docs/components/datastore.rst

```diff
@@ -1,169 +1,79 @@
-DKAN Datastore
+Datastore
 ===============
 
```

Comment:
This opening bit reads a bit rough, can we start with how resources work, how the datastore enhances usability, and why it is good to take advantage of the option.

```
When you create a dataset with resources, DKAN is reading the data directly from the resource file or API.

The Datastore component provides an option for you to parse CSV files and save the data into database tables. This allows users to query the data through a public API.

So by adding your CSV resources to the datastore, you are getting the fullest functionality possible out of your datasets.
```